stories/core/Buttons/ButtonGroup.stories.tsx

```diff
@@ -37,3 +54,61 @@ export const WithIcons: Story<ButtonGroupProps> = () => {
     </ButtonGroup>
   );
 };
+
+export const Overflow: Story<ButtonGroupProps> = (args) => {
+  const buttons = Array(10)
```

Comment:
I am thinking.. In real use case there wont be any buttons array. Usually they just drop one by one buttons in there. And if we return index, we kinda force them to change that and put all the buttons in array...

Reply:
If they want to menu items, they need an array somewhere